src/panel/TemplateEditor/Sidebar/PortalDialog.tsx

```diff
@@ -31,7 +32,24 @@ export default ({ open, onClose }: Props) => {
         template: templateFile,
         url: inspectedWindowUrl
     }).toString();
-    const url = `${portal?.views?.full}?${params}`;
+    const [ embedUrl, setEmbedUrl ] = useState<string>();
+    
+    useEffect(() => {
```

Comment:
Move this into a new PortalContext within the TemplateEditor context such that `dialogUrl` comes from `usePortal`.